xml/common_intro_feedback_i.xml

```diff
@@ -15,22 +15,23 @@
  </info>
 
  <para>
-  Your feedback and contributions to this documentation are welcome! Several
```

Comment:
To be honest, I think the old sentence was easier to read than the new one with the insertions between the commas.

Reply:
fixed as suggested with minor improvement of 2nd sentence